pkg/booleanpolicy/violationmessages/printer/container.go

```diff
@@ -157,40 +157,36 @@ func imageUserPrinter(fieldMap map[string][]string) ([]string, error) {
 
 const (
 	imageSignatureVerifiedTemplate = `{{if .ContainerName}}Container '{{.ContainerName}}' image` +
-		`{{else}}Image{{end}} signature is {{.Status}}`
+		`{{else}}Image{{end}} signature is not verified by the specified signature integration(s)` +
+		`{{if .SuccessfulVerifiersRepr}} (it is verified by other integration(s): {{ .SuccessfulVerifiersRepr }}){{end}}.`
 )
 
+// imageSignatureVerifiedPrinter returns a violation message explaining that the specified signature integration could
+// not verify any image signature. If other signature integrations did verify the image, they are listed in the message.
```

Comment:
"could not verify" or "hasn't verified"? If indeed the former, the template message should say "signature could not be verified".

Reply:
Here: https://github.com/stackrox/stackrox/pull/16135/commits/8ff79b3196f99385ab89c3c408540a113657feec